### ISSUE TITLE
YOLOv8: Model architecture prints over and over.

### ISSUE THREAD

**Issue by Vaevin** [ultralytics/ultralytics#988]
During training, the model architecture overview keeps getting reprinted to the console over and over every epoch.

`from ultralytics import YOLO

model = YOLO("yolov8n.yaml")

if __name__ == '__main__':  
    model.train(data="whole_iris.yaml", epochs=10, device=0, batch = 4, save=True, imgsz=640)`

![image](https://user-images.githubusercontent.com/60018242/219066119-dcec0044-6dab-4805-bcc4-e86912d0769f.png)

**Comment by Laughing-q** [comment]
@joshleiph can't reproduce your issue, can you upgrade the package and try again?
![image](https://user-images.githubusercontent.com/61612323/219249616-908bb0c5-3d83-4ee6-8e6a-551ef8924f4e.png)

**Comment by mustafacavusoglu** [comment]
I just downloaded yolov8. Same issue

**Comment by AyushExel** [comment]
I can't reproduce it either.

**Comment by mustafacavusoglu** [comment]
there is no issue like this when I train on jupyter notebook . But it occurs when I train on cmd (windows 10)

**Comment by joshleiph** [comment]
@Laughing-q I've made sure my repo is up to date, and I have installed all the requirements.txt modules. 

Some additional information:
Some of my training images/labels that are part of my dataset are "corrupt". However, I do not believe this should have any effect on repeated model print outs.
![image](https://user-images.githubusercontent.com/60018242/219417005-a88f73bf-eb4c-4b99-b0da-7c3331661c42.png)

**Comment by Laughing-q** [comment]
@joshleiph @mustafacavusoglu please also provide the command that we can reproduce, or you can try `data=coco128.yaml` to see if this still happens.

**Comment by mustafacavusoglu** [comment]
in windows 10:
      - conda activate v8
      - python train.py
     
from ultralytics import YOLO

data_path = './mlp_cc/custom_data.yaml'
model = YOLO(r"ultralytics\models\v8\yolov8n.yaml")
cfg = {
    'device' : 0,
    'imgsz' : 512,
    'device' : 0,
    'epochs' : 10,
    'patience' : 0,
    'cache' : False,
    'batch' : 2,
    'name' : 'v8_train',
    'verbose' : False
}


if __name__ == '__main__':
    # Use the model
    results = model.train(data=data_path, **cfg)  # train the model
    metrics = model.val()

**Comment by mustafacavusoglu** [comment]
in windows 10:
      - conda activate v8
      - python train.py
```{python}
from ultralytics import YOLO

data_path = './mlp_cc/custom_data.yaml'
model = YOLO(r"ultralytics\models\v8\yolov8n.yaml")
cfg = {
    'device' : 0,
    'imgsz' : 512,
    'device' : 0,
    'epochs' : 10,
    'patience' : 0,
    'cache' : False,
    'batch' : 2,
    'name' : 'v8_train',
    'verbose' : False
}

if __name__ == '__main__':
    # Use the model
    results = model.train(data=data_path, **cfg)  # train the model
    metrics = model.val()
```

**Comment by mustafacavusoglu** [comment]
> 

In windows 10 it is same for coco128.yaml. prints model architecture  in each epoch

**Comment by joshleiph** [comment]
I believe I have resolved the problem. When I initiated the `YOLO(model=)` instance, instead of passing:

`model = YOLO(model = "yolov8n.yaml")`

I instead used the weights (.pt) from the first original training:

`model = YOLO(model = "best.pt")`

where best.pt is the weights from my original YOLOv8 training using the yolov82.yaml architecture.

**Comment by Vaevin** [comment]
This problem still occurs when I simply run:

`from ultralytics import YOLO

model_path = "yolov8n.yaml"
model = YOLO(model=model_path)
dataset = "whole_iris.yaml"


if __name__ == '__main__':  
    model.train(data=dataset, epochs=10, device=0, batch = 8, save=True, imgsz=640)`

The whole_iris.yaml file contains:
`# Train/val/test sets as 1) dir: path/to/imgs, 2) file: path/to/imgs.txt, or 3) list: [path/to/imgs1, path/to/imgs2, ..]
path: ../datasets/Whole_Iris # dataset root dir. This will be in the datasets folder (parent folder of this repo).
train: images/train   # train images (relative to 'path')
val: images/val       # val images (relative to 'path')
test: images/test     # test images (relative to 'path')

# number of classes
nc: 1

# class names
names:
  0: Iris`

Any idea why the model archietecture would be printed out over and over like this? @Laughing-q
